pkg/minikube/schedule/daemonize_windows.go

```diff
@@ -20,16 +20,93 @@ package schedule
 
 import (
 	"fmt"
+	"os/exec"
 	"time"
 
+	"github.com/pkg/errors"
 	"k8s.io/klog/v2"
+	"k8s.io/minikube/pkg/minikube/assets"
+	"k8s.io/minikube/pkg/minikube/constants"
+	"k8s.io/minikube/pkg/minikube/machine"
+	"k8s.io/minikube/pkg/minikube/sysinit"
 )
 
 // KillExisting will kill existing scheduled stops
 func KillExisting(profiles []string) {
-	klog.Errorf("not yet implemented for windows")
+	for _, profile := range profiles {
+		if err := killExisting(profile); err != nil {
+			klog.Errorf("error terminating scheduled stop for profile %s: %v", profile, err)
+		}
+	}
 }
 
+func killExisting(profile string) error {
+	klog.Infof("trying to kill existing schedule stop for profile %s...", profile)
+	api, err := machine.NewAPIClient()
+	if err != nil {
+		return errors.Wrapf(err, "getting api client for profile %s", profile)
+	}
+	h, err := api.Load(profile)
+	if err != nil {
+		return errors.Wrap(err, "Error loading existing host. Please try running [minikube delete], then run [minikube start] again.")
+	}
+	runner, err := machine.CommandRunner(h)
+	if err != nil {
+		return errors.Wrap(err, "getting command runner")
+	}
+	// restart scheduled stop service in container
+	sysManger := sysinit.New(runner)
+	if err := sysManger.Stop(constants.ScheduledStopSystemdService); err != nil {
```

Comment:
what happens when error happens ? do we delete from config?

Reply:
updated to only save config if daemonize succeeds